mne_bids/copyfiles.py

```diff
@@ -17,12 +17,15 @@
 import os
 import os.path as op
 import re
+import datetime
 import shutil as sh
 
+from mne.io import read_raw_brainvision, anonymize_info
 from scipy.io import loadmat, savemat
 
 from mne_bids.read import _parse_ext
 from mne_bids.utils import _get_mrk_meas_date
+from mne_bids.write import _check_anonymize
```

Comment:
isn't this in utils?

Reply:
yes O.o ... how/why are tests passing?

Reply:
no idea why tests were passing, but I fixed this.